website/docs/docs/deploy/deployment-tools.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Deploy with other tools"
+id: "deployment-tools"
+sidebar: "Deploy with other tools"
+---
+
+Discover effortless ways to schedule and run your dbt jobs with the help of powerful tools like [dbt Cloud](/docs/deploy/dbt-cloud-job), Airflow, Prefect, Dagster, automation server, Cron, or Azure Data Factory (ADF). 
+
+You can use these tools to save time and increase efficiency by automating your data workflows, all while enjoying a hassle-free experience. 
+
+
+## Airflow
+
+If your organization is using [Airflow](https://airflow.apache.org/), there are a number of ways you can run your dbt jobs, including:
+
+<Tabs>
+
+<TabItem value="airflowcloud" label="dbt Cloud">
+
+Installing the [dbt Cloud Provider](https://registry.astronomer.io/providers/dbt-cloud) to orchestrate dbt Cloud jobs. This package contains multiple Hooks, Operators, and Sensors to complete various actions within dbt Cloud.
+
+<Lightbox src="/img/docs/running-a-dbt-project/airflow_dbt_connector.png" title="Airflow DAG using DbtCloudRunJobOperator"/>
+<Lightbox src="/img/docs/running-a-dbt-project/dbt_cloud_airflow_trigger.png" title="dbt Cloud job triggered by Airflow"/>
+
+</TabItem>
+
+<TabItem value="airflowcore" label="dbt Core">
+
+Invoking dbt Core jobs through the [BashOperator](https://registry.astronomer.io/providers/apache-airflow/modules/bashoperator). In this case, be sure to install dbt into a virtual environment to avoid issues with conflicting dependencies between Airflow and dbt.
+
+</TabItem>
+</Tabs>
+
+For more details on both of these methods, including example implementations, check out [this guide](https://www.astronomer.io/guides/airflow-dbt).
+
+## Azure Data Factory
```

Comment:
@kbrock91 - have added adf info here as written steps and your video too! let me know what you think!